### PR TITLE
Route branch

### PR DESCRIPTION
**Pull request by soesandarwin2201**
## What does this PR do?
This PR is raised to l create a full website for the Math magicians app, consisting of several pages and using the components I already created.

## Description of the task to be completed
- [x] create the route-branch 
- [x] install the react-router-dom
- [x] create the home page, quotes page and navbar components
- [x]  use routes in the app.js
- [x] test the calculator
- [x]  check the linter errors  
## How should it be tested
Clone the repository
```
git clone  https://github.com/soesandarwin2201/React-Project
```
Navigate to the folder
```
cd React-Project
```
Checkout the branch
```
git checkout route-branch
```
#### Thanks for reviewing my PR :blush:

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -18,82 +18,89 @@ const  Claculator = () =>  {
 
    const { total, next, operation } = this.state;
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] When I navigate to the Calculator page the app crash and goes blank. Kindly look into that.

  app when I navigate to calculator route | error message in console
  :---:|:---:
  didioe | errir

  You can fix this by removing every occurrence of `this.` in this component. You need to change this:
  ```
  const { total, next, operation } = this.state;
  ```
  to this:
  ```
  const { total, next, operation } = state;
  ```

  Also change every `this.eventHandler` to this `eventHandler`

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @soesandarwin2201,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (soesandarwin2201) [reply src/components/Calculator.js]: thanks for your suggestions and also for providing useful information for me. The issue was when I resolved the conflict, might happen that error, I will try to solve that now. And Again thanks for your time and effort.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @soesandarwin2201, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**